src/app/FakeLib/DotNetCLIHelper.fs

```diff
@@ -1,5 +1,5 @@
 ﻿/// Contains a task which can be used to run dotnet CLI commands.
-module Fake.DotNet
+module Fake.DotNetCli
```

Comment:
They changed name after beta, 6 months ago. 

I think we should try to use were possible to use the correct name. 
That help new users, because it's easier to search docs. 
That said I liked more dotnet cli a lot as name. 

For example we should stop to say coreclr when we mean .net core ( coreclr+corefx).